### PR TITLE
MT check if a modelless projectiles should be drawn. 

### PR DESCRIPTION
**Pull request by lhog**
Partial fix for https://github.com/beyond-all-reason/spring/issues/1202

### CODE REVIEW

File: rts/Rendering/Env/Particles/ProjectileDrawer.cpp [hunk 1]
```diff
@@ -619,10 +629,12 @@ void CProjectileDrawer::DrawProjectileNow(CProjectile* pro, bool drawReflection,
 		return;
 
 	// no-op if no model
-	DrawProjectileModel(pro);
+	if (DrawProjectileModel(pro))
```
Comment (sprunk) [inline rts/Rendering/Env/Particles/ProjectileDrawer.cpp:632]: Could this be a standalone change? It looks unrelated to all the MT stuff so could get in independently.

Comment (lhog) [reply rts/Rendering/Env/Particles/ProjectileDrawer.cpp:632]: I didn't test it extensively, but it looked like an inherited spring bug.
Maybe it can go standalone indeed.
